cw-orch/src/snapshots.rs

```diff
@@ -1,30 +1,41 @@
 //! Defined the snapshot testing macro on the Mock environment
 //! This is included here and not in the mock package because it needs to import traits to work
 
+/// Function helper used to parse storage into readable strings
+pub fn parse_storage(storage: Vec<(Vec<u8>, Vec<u8>)>) -> Vec<(String, String)> {
```

Comment:
nit: this function doesn't benefit from ownership of the storage

Reply:
Corrected